<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-G1QB867" failures="1" tests="1" name="Test" time="1.754" errors="0" timestamp="2025-04-25T21:17:00 IST">
  <testcase classname="testscript.Basetest" name="tc01" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.NullPointerException: Cannot invoke &amp;quot;org.apache.poi.ss.usermodel.Row.getCell(int)&amp;quot; because &amp;quot;row&amp;quot; is null">
      <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Row.getCell(int)" because "row" is null
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Row.getCell(int)" because "row" is null
at testscript.Excel_util.readExcelData(Excel_util.java:32)
at testscript.Basetest.getExcelData(Basetest.java:31)
... 25 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- tc01 -->
</testsuite> <!-- Test -->
